src/test/common/test_time.cc

```diff
@@ -176,3 +177,42 @@ TEST(TimePoints, SignedSubtraciton) {
   ASSERT_GT(cmtb - cmta, ceph::signedspan::zero());
   ASSERT_GT((cmtb - cmta).count(), 0);
 }
+
+TEST(SteadyClock, DurationCastFromDoubleSeconds) {
+  mono_clock::duration d;
+  std::chrono::duration<double> d_double_secs(0.000123);
+
+  d = std::chrono::duration_cast<mono_clock::duration>(d_double_secs);
```

Comment:
i am against these new tests for `mono_clock`. but IMO, the major part of the functionalities being tested here are part of `std::chrono`. so it seems that we are building a test suite for `std::chrono` libstdc++ or libc++ ?

@adamemerson what do you think?

Reply:
I understand, however, internal changes to `ceph::mono_clock` might break some of these tests, which in turn might alter Ceph's behavior in subtle ways. I believe I added this specific test to make sure the measured duration was a double when comparing it to a floating point value read from `g_conf` (options.cc). If you think they're out of scope or too trivial I'd be happy to remove them.

Reply:
I agree with @tchaikov I think most of these are out of scope, since the functionality basically just relies on us actually implementing std::chrono

Reply:
As above, I'm not sure that having our own clock really gets us something here.